### PR TITLE
Add translations to French

### PR DESCRIPTION
**Pull request by sergeicodes**
Translated by @MarionCouesnon.

- [x] Add visually hidden text to "read more" links
- [x] Check bold highlights
- [x] Update English version where needed
- [x] Create issues for updating translations
- [x] Add `og:image`

### CODE REVIEW

File: fr/index.html [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+              <a class="nav__link" lang="pt-br" href="../pt-br/">Português (Brasil)</a>
+            </li>
+            <li class="nav__list-item">
+              <a class="nav__link" lang="pt" href="../pt/">Português (Portugal)</a>
+            </li>
+          </ul>
+        </nav>
+
+        <h1>Les mythes de l’accessibilité</h1>
+        <p class="stamp">Debunked</p>
```
Comment (sergeicodes) [inline fr/index.html]: @MarionCouesnon Could you translate the word `Debunked`

File: fr/index.html [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+            C'est un travail d'équipe. Les <strong>designers</strong> créent des interfaces utilisateur accessibles, les
+            <strong>développeurs</strong> les construisent, les <strong>ingénieurs AQ</strong> effectuent les tests
+            d'accessibilité, les responsables <strong>de projet</strong> s'assurent que l'accessibilité est incluse dans
+            les processus de l'équipe, l'<strong>équipe juridique</strong> vérifie qu'un produit ne présente aucun
+            risque du point de vue de l'accessibilité, les <strong>gestionnaires de contenu</strong> adaptent le contenu
+            pour qu'il soit conforme.
+          </p>
+
+          <a href="https://accessibility.digital.gov/" target="_blank"
+            >Plus de détails <span class="visually-hidden"></span
```
Comment (sergeicodes) [inline fr/index.html]: @MarionCouesnon could you please add a myth name to `visually-hidden`.

File: fr/index.html [hunk 3]
```diff
@@ -0,0 +1,391 @@
...
+          <h2 class="myth__title">L'accessibilité ne concerne que les utilisateurs aveugles</h2>
+          <p>
+            <strong>Certes, l’accessibilité agit sur l’expérience des</strong> utilisateurs aveugles mais aussi sur
+            celle des utilisateurs sourds, des utilisateurs ayant des handicaps moteurs et cognitifs... Elle agit aussi
+            sur celle des utilisateurs âgés; bref, de tous les utilisateurs. Elle est essentielle pour certains mais
+            utile pour tous.
+          </p>
+
+          <a href="https://www.w3.org/WAI/fundamentals/accessibility-intro/" target="_blank"
+            >Plus de détails <span class="visually-hidden"></span
```
Comment (sergeicodes) [inline fr/index.html]: @MarionCouesnon could you please add a myth name to `visually-hidden`.

File: fr/index.html [hunk 4]
```diff
@@ -0,0 +1,391 @@
...
+        <section class="myth">
+          <h2 class="myth__title">Nous n'avons pas besoin d'être conformes</h2>
+
+          <p>
+            Peut-être que vous en avez besoin. Il existe un certain nombre de lois et de règlements qui peuvent exiger
+            qu'un site Web respecte les principes d'accessibilité. Cela vaut assurément la peine de vérifier.
+          </p>
+
+          <a href="https://www.w3.org/WAI/policies/" target="_blank"
+            >Plus de détails <span class="visually-hidden"></span
```
Comment (sergeicodes) [inline fr/index.html]: @MarionCouesnon could you please add a myth name to `visually-hidden`.

File: fr/index.html [hunk 5]
```diff
@@ -0,0 +1,391 @@
...
+          </h2>
+
+          <p>
+            Non. L'entretien quotidien, les nouveaux collègues qui n'ont pas les compétences requises en matière
+            d'accessibilité, les nouvelles exigences, tout cela peut compromettre l'accessibilité. L'accessibilité est
+            un élément incontournable qui doit être pris en compte à chaque étape du développement.
+          </p>
+
+          <a href="https://ethanmarcotte.com/wrote/accessibility-is-not-a-feature/" target="_blank"
+            >Plus de détails <span class="visually-hidden"></span
```
Comment (sergeicodes) [inline fr/index.html]: @MarionCouesnon could you please add a myth name to `visually-hidden`.

File: fr/index.html [hunk 6]
```diff
@@ -0,0 +1,391 @@
...
+
+          <p>
+            C'est faux. D’autant plus que dans certains cas, Internet est le seul moyen pour les personnes handicapées
+            de communiquer avec le monde. Là où les “personnes sans handicap” utilisent leurs facultés innées telles que
+            la vue et l'ouïe, les personnes handicapées recourent à des technologies d’assistance pour avoir accès aux
+            mêmes contenus.
+          </p>
+
+          <a href="https://www.w3.org/WAI/people-use-web/" target="_blank"
+            >Plus de détails <span class="visually-hidden"></span
```
Comment (sergeicodes) [inline fr/index.html]: @MarionCouesnon could you please add a myth name to `visually-hidden`.

File: fr/index.html [hunk 7]
```diff
@@ -0,0 +1,391 @@
...
+        <section class="myth">
+          <h2 class="myth__title">Les personnes aveugles ne regardent pas de films</h2>
+
+          <p>
+            C'est tout à fait faux. Grâce aux audio-descriptions, ils peuvent regarder et écouter n'importe quel contenu
+            multimédia comme tout le monde.
+          </p>
+
+          <a href="https://www.w3.org/WAI/media/av/description/" target="_blank"
+            >Plus de détails <span class="visually-hidden"></span
```
Comment (sergeicodes) [inline fr/index.html]: @MarionCouesnon could you please add a myth name to `visually-hidden`.

File: fr/index.html [hunk 8]
```diff
@@ -0,0 +1,391 @@
...
+          <h2 class="myth__title">L'accessibilité ne peut être testée que par des personnes handicapées</h2>
+
+          <p>
+            En général, les personnes handicapées sont les mieux placées pour effectuer des tests d’accessibilité, car
+            elles utilisent des technologies d'assistance tout le temps. Cependant, tout le monde peut apprendre à
+            tester l'accessibilité des sites Web.
+          </p>
+
+          <a href="https://www.w3.org/WAI/test-evaluate/" target="_blank"
+            >Plus de détails <span class="visually-hidden"></span
```
Comment (sergeicodes) [inline fr/index.html]: @MarionCouesnon could you please add a myth name to `visually-hidden`.

File: fr/index.html [hunk 9]
```diff
@@ -0,0 +1,391 @@
...
+    <title>Les mythes de l’accessibilité</title>
+    <link rel="stylesheet" href="../css/styles.css" />
+    <link rel="icon" type="image/x-icon" href="../favicon.ico" />
+
+    <meta name="description" content="A small project debunking common accessibility myths." />
+    <meta name="image" content="https://a11ymyths.com/" />
+
+    <meta property="og:title" content="Les mythes de l’accessibilité" />
+    <meta property="og:description" content="A small project debunking common accessibility myths." />
+    <meta property="og:image" content="https://a11ymyths.com/images/og-image.png" />
```
Comment (sergeicodes) [inline fr/index.html]: Add `og:image` for French once the word `Debunked` is translated.

File: fr/index.html [hunk 10]
```diff
@@ -0,0 +1,391 @@
...
+    <meta property="og:image:alt" content="Les mythes de l’accessibilité" />
+    <meta property="og:url" content="https://a11ymyths.com/" />
+
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:creator" content="@_sergeikriger" />
+    <meta name="twitter:title" content="Les mythes de l’accessibilité" />
+    <meta name="twitter:description" content="A small project debunking common accessibility myths." />
+    <meta name="twitter:image" content="https://a11ymyths.com/images/og-image.png" />
```
Comment (sergeicodes) [inline fr/index.html]: Add `og:image` for French once the word `Debunked` is translated.

File: fr/index.html [hunk 11]
```diff
@@ -0,0 +1,391 @@
...
+<html lang="fr">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Les mythes de l’accessibilité</title>
+    <link rel="stylesheet" href="../css/styles.css" />
+    <link rel="icon" type="image/x-icon" href="../favicon.ico" />
+
+    <meta name="description" content="A small project debunking common accessibility myths." />
```
Comment (sergeicodes) [inline fr/index.html]: @MarionCouesnon Could you translate this phrase.

File: fr/index.html [hunk 12]
```diff
@@ -0,0 +1,391 @@
...
+    <meta name="image" content="https://a11ymyths.com/" />
+
+    <meta property="og:title" content="Les mythes de l’accessibilité" />
+    <meta property="og:description" content="A small project debunking common accessibility myths." />
```
Comment (sergeicodes) [inline fr/index.html]: @MarionCouesnon Could you translate this phrase.

File: fr/index.html [hunk 13]
```diff
@@ -0,0 +1,391 @@
...
+    <meta property="og:image" content="https://a11ymyths.com/images/og-image.png" />
+    <meta property="og:image:alt" content="Les mythes de l’accessibilité" />
+    <meta property="og:url" content="https://a11ymyths.com/" />
+
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:creator" content="@_sergeikriger" />
+    <meta name="twitter:title" content="Les mythes de l’accessibilité" />
+    <meta name="twitter:description" content="A small project debunking common accessibility myths." />
```
Comment (sergeicodes) [inline fr/index.html]: @MarionCouesnon Could you translate this phrase.

File: fr/index.html (hunk 12 above)
Comment (MarionCouesnon) [reply fr/index.html]: "Un petit projet qui déboulonne les mythes les plus répandus sur l'accessibilité"

File: fr/index.html (hunk 13 above)
Comment (MarionCouesnon) [reply fr/index.html]: Done with 646f61ff26a73ba84f69468cafc139a12f7baf24

File: fr/index.html (hunk 12 above)
Comment (MarionCouesnon) [reply fr/index.html]: Done with 646f61ff26a73ba84f69468cafc139a12f7baf24

File: fr/index.html (hunk 11 above)
Comment (MarionCouesnon) [reply fr/index.html]: Done with 646f61ff26a73ba84f69468cafc139a12f7baf24

File: fr/index.html (hunk 5 above)
Comment (MarionCouesnon) [reply fr/index.html]: Done with 77c2a05d2459067877887858494997c996167cdd

File: fr/index.html (hunk 4 above)
Comment (MarionCouesnon) [reply fr/index.html]: Done with 77c2a05d2459067877887858494997c996167cdd

File: fr/index.html (hunk 3 above)
Comment (MarionCouesnon) [reply fr/index.html]: Done with 77c2a05d2459067877887858494997c996167cdd

File: fr/index.html (hunk 2 above)
Comment (MarionCouesnon) [reply fr/index.html]: Done with 77c2a05d2459067877887858494997c996167cdd

**Pull request merged**